rts/lra/coordinator/src/test/java/io/narayana/lra/coordinator/domain/model/LRATest.java

```diff
@@ -904,20 +904,20 @@ private static String makeLink(String uriPrefix, String key) {
                 .build().toString();
     }
 
-    private void clearObjectStore() {
+    public static void clearObjectStore(TestName testName) {
         final String objectStorePath = arjPropertyManager.getObjectStoreEnvironmentBean().getObjectStoreDir();
         final File objectStoreDirectory = new File(objectStorePath);
 
-        clearDirectory(objectStoreDirectory);
+        clearDirectory(objectStoreDirectory, testName);
     }
 
-    private void clearDirectory(final File directory) {
+    public static void clearDirectory(final File directory, TestName testName) {
```

Comment:
+1 to the need for this testName parameter given that the clearDirectory function is essentially being proposed as a utility method for use outside of LRATest class now and because testName is already used in here in the existing implementation of the method.

Reply:
I think that is a good idea, I will use a common test base class then